tests/client/core/reducers/test_authentication.js

```diff
@@ -27,4 +30,25 @@ describe('authentication reducer', () => {
       auth({ token: 'foo' }, { type: 'SET_CURRENT_USER', payload: { username } }),
       { token: 'foo', username });
   });
+
+  describe('LOG_OUT_USER', () => {
+    let remove;
+
+    before(() => {
+      remove = sinon.stub(cookie, 'remove');
+      sinon.stub(config, 'get').returns('JWT_COOKIE_NAME');
+    });
+
+    it('clears the state', () => {
+      assert.deepEqual(
+        auth({ token: 'hey!', otherThing: 'goes away' }, { type: 'LOG_OUT_USER' }),
+        {});
+    });
+
+    it('clears the cookie', () => {
+      auth({ token: 'hey!', otherThing: 'goes away' }, { type: 'LOG_OUT_USER' });
+      assert.ok(remove.called);
+      assert.ok(remove.calledWith('JWT_COOKIE_NAME'));
```

Comment:
I think I meant to use `calledOnce` but it seems unlikely that this would be called multiple times. I'll take it out.